Sources/SentrySwiftUI/SentryTracedView.swift

```diff
@@ -34,41 +34,61 @@ import SentryInternal
 ///         //The part of your content you want to measure
 ///     }.sentryTrace("My Awesome Screen")
 ///
-///
 @available(iOS 13, macOS 10.15, tvOS 13, watchOS 6.0, *)
 public struct SentryTracedView<Content: View>: View {
-    
+    @State var viewWasPresented = false
+
     let content: () -> Content
     let name: String
-    let id: SpanId
-    
+    let nameSource: SentryTransactionNameSource
+
     public init(_ viewName: String? = nil, content: @escaping () -> Content) {
         self.content = content
         self.name = viewName ?? SentryTracedView.extractName(content: Content.self)
-        id = SentryPerformanceTracker.shared.startSpan(withName: self.name,
-                                                       nameSource: viewName == nil ? .component : .custom,
-                                                       operation: "ui.load")
+        self.nameSource = viewName == nil ? .component : .custom
     }
-    
+
     private static func extractName(content: Any) -> String {
         var result = String(describing: content)
-        
+
         if let index = result.firstIndex(of: "<") {
             result = String(result[result.startIndex ..< index])
         }
-        
+
         return result
     }
-    
+
     public var body: some View {
+        if viewWasPresented {
```

Comment:
`h`: I don't see any tests being updated that reflect these changes or am I missing something?

Reply:
Added an UITest to ensure viewWasPresented is used